lib/notify.c

```diff
@@ -105,6 +105,7 @@ system_call_script(thread_master_t *m, int (*func) (thread_t *), void * arg, lon
 
 	/* In case of this is parent process */
 	if (pid) {
+		setpgid(pid, pid);
```

Comment:
Oops... I'm sorry!
It's completely my mistake.

I agree with you. It should be setpgid() is child process.